app/setup.py

```diff
@@ -376,24 +392,24 @@ def setup_publisher(application):
         application.eq["publisher"] = LogPublisher()
 
     else:
-        raise Exception("Unknown EQ_PUBLISHER_BACKEND")
+        raise UnknownEnviormentVariable("Unknown EQ_PUBLISHER_BACKEND")
 
 
 def setup_feedback(application):
     if application.config["EQ_FEEDBACK_BACKEND"] == "gcs":
-        bucket_name = application.config.get("EQ_GCS_FEEDBACK_BUCKET_ID")
-
-        if not bucket_name:
-            raise Exception("Setting EQ_GCS_FEEDBACK_BUCKET_ID Missing")
-
-        application.eq["feedback_submitter"] = GCSFeedbackSubmitter(
-            bucket_name=bucket_name
-        )
+        if bucket_name := application.config.get("EQ_GCS_FEEDBACK_BUCKET_ID"):
```

Comment:
Same s above.

Reply:
Changed back to the old version but refactored to use walrus operator 👍